openqa-label-known-issues

```diff
@@ -5,8 +5,10 @@
 
 set -o pipefail -o errtrace
 
+dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+PATH=$dir:$PATH
```

Comment:
What is that needed for?

Reply:
When the `test/07-openqa-label-known-issues.t` runs tries and tries `source openqa-label-known-issues`  and it cant find the `_common`

Reply:
Try what we do in `openqa-investigate`:

```suggestion
. "$(dirname "${BASH_SOURCE[0]}")"/_common 
```

Reply:
I tried. And Unfortunately breaks some tests. I am not sure it is necessary. or.. is it?

Reply:
I tried it as well, and it worked for me

Reply:
like I wrote below, you don't want to set `dir` here, because it is set by `test/init`. I see you are doing it to use it in the test, but it is not necessary, and confusing.
I don't know how to explain it better.

Reply:
00a7075